modules/images/webp-uploads/load.php

```diff
@@ -704,3 +704,187 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 	return rest_ensure_response( $data );
 }
 add_filter( 'rest_prepare_attachment', 'webp_uploads_update_rest_attachment', 10, 3 );
+
+/**
+ * Inspect if the current call to `wp_update_attachment_metadata()` was done from within the context
+ * of an edit to an attachment either restore or other type of edit, in that case we perform operations
+ * to save the sources properties, specifically for the `full` size image due this is a virtual image size.
+ *
+ * @since n.e.x.t
+ *
+ * @see wp_update_attachment_metadata()
+ *
+ * @param array $data The current metadata of the attachment.
+ * @param int   $attachment_id The ID of the current attachment.
```

Comment:
Please indent parameter docs correctly, so that they are aligned:
```suggestion
 * @param array $data          The current metadata of the attachment.
 * @param int   $attachment_id The ID of the current attachment.
```

Please also do this for the other function doc comments in this PR.